smoke_test/config.py

```diff
@@ -32,12 +32,10 @@
 ASG = PATH_DIR_NAME / 'asg-hevc'
 FEI_EXTRACTOR = PATH_DIR_NAME / 'hevc_fei_extractor'
 SAMPLE_FEI = Path('/opt/intel/mediasdk/samples/sample_hevc_fei')
```

Comment:
Developers merged the commit which changed default layout.
New path is: `/opt/intel/mediasdk/share/mfx/samples`
(but backwards compatibility is also needed)